### PR TITLE
Fetch passwords from secretsmanager

### PR DESCRIPTION
**Pull request by seve**
#### Reviewers
**Functional:** @tihuan 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

File: client/__tests__/globalSetup.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+module.exports = async () => {
+  await setup();
+  try {
+    const secret = JSON.parse((await client.send(command)).SecretString);
+    process.env.TEST_ACCOUNT_PASS = secret.test_account_password;
+  } catch (error) {
+    console.error(error);
+  }
+  console.log(process.env.TEST_ACCOUNT_PASS);
```
Comment (tihuan) [inline client/__tests__/globalSetup.js:24]: rogue console.log that can be deleted 😆

File: client/__tests__/e2e/config.js [hunk 2]
```diff
@@ -6,3 +6,5 @@ export const appUrlBase =
 export const DATASET = "pbmc3k";
 export const isDev = jestEnv === ENV_DEFAULT.DEV;
 export const isDebug = jestEnv === ENV_DEFAULT.DEBUG;
+export const TEST_EMAIL = "user@example.com";
```
Comment (tihuan) [inline client/__tests__/e2e/config.js:9]: Ah `user@example.com` is only available in staging, so we'll need to create the same account in prod Auth0 tenant!

**Review by tihuan — COMMENTED** [review]
Uhh [this PR](https://github.com/chanzuckerberg/cellxgene/pull/2140/files) adds 29651 LoC back, so 29651 - 22693 = net +6958 LoC. Not sure if that's "better" 😂

File: client/__tests__/globalSetup.js (hunk 1 above)
Comment (seve) [reply client/__tests__/globalSetup.js:24]: Nice catch TY!

**Pull request merged**